gaurun/client.go

```diff
@@ -5,7 +5,7 @@ import (
 	"net/http"
 	"time"
 
-	"github.com/mercari/gcm"
+	"github.com/mercari/gaurun/service/google"
```

Comment:
> there is not strong reason

Strong reason is it's no more only `gcm` and `gcm` will be replaced with fcm in future.